game/controllers.hh

```diff
@@ -61,11 +61,13 @@ namespace input {
 		HWButton hw; ///< Hardware button number (for internal use and debugging only)
 		Button button; ///< Mapped button id
 		NavButton nav; ///< Navigational button interpretation
-		std::optional<double> value; ///< false for button release, up to 1.0 for press (e.g. velocity value), or axis value (-1.0 .. 1.0)
+		std::optional<double> val; ///< false for button release, up to 1.0 for press (e.g. velocity value), or axis value (-1.0 .. 1.0)
+		double value() const { return val.value_or(0.0); }
+		uint32_t uValue() const { return static_cast<uint32_t>(value()); }
```

Comment:
Personally i wouldn't use the fixed-width type. Since it doesn't solve the problem we're trying to tackle (conversion warnings).
As said on discord fixed-width types are a thing when you do embedded stuff and need to exactly know how many bytes a certain type is since sometimes you need to do micro optimizations like that for your specific hardware/compiler. 

For us that's not the case, so `int` should suffice here.
Also not going to repeat myself throughout this review, so please change it accordingly in other places

Reply:
It actually should solve some of them, if you are careful to (where possible) always mix unsigneds with bigger signed types, which is what I tried to do in a few places.

Reply:
the code was removed;no more int32_t on last version